index.html

```diff
@@ -6,6 +6,11 @@
     <title>Document</title>
 </head>
 <body>
-    STUFFFFF
+    <main>
+        <section id="hero">
+            <h1>WOW a picture!</h1>
```

Comment:
it would be nice to use a descriptive element for the images as well such as a figure element:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure